### ISSUE TITLE
To include a callback to transfer the list of applications subscribed to receive the embedding status change(/v1/notify-embedding-status-changes) from old version to new version  

### ISSUE THREAD

**Issue by PrathibaJee** [openBackhaul/RegistryOffice#530]
As per the specification , /v1/bequeath-your-data-and-die will transfer the following data to the new release , 
- registered applications (via NR://v2/register-application) 
- application that approves (via NR:///v1/inquire-application-type-approvals)

In the RegistryOffice version 2.1.0 , we have a new API /v1/notify-embedding-status-changes using which applications shall subscribe for 
embedding status change notifications.

**Proposal :**
To transfer the details of the applications subscribed for the embedding status change , shall we include a callback ?
If so , a callback needs to be included in the /v1/bequeath-your-data-and-die to do this transfer.

**Comment by openBackhaul** [comment]
Yes, transfer of subscriptions for embedding status changes to NewRelease shall be facilitated.

ServiceList:
- OperationClient /v1/notify-embedding-status-changes (ro-2-1-0-op-c-im-nr-2-1-0-006) has been added to the NewRelease

ForwardingList:
- UUID of existing forwarding PromptForBequeathingDataCausesTransferOfListOfAlreadyRegisteredApplications has changed from ro-2-1-0-op-fc-im-114 to ro-2-1-0-op-fc-im-112
- New forwarding PromptForBequeathingDataCausesTransferOfListOfSubscriptionsForEmbeddingStatusChanges (ro-2-1-0-op-fc-im-113) has been added

CONFIGfile:
- OperationClient /v1/notify-embedding-status-changes (ro-2-1-0-op-c-im-nr-2-1-0-006) has been added to the NewRelease
- UUID of existing forwarding PromptForBequeathingDataCausesTransferOfListOfAlreadyRegisteredApplications has changed from ro-2-1-0-op-fc-im-114 to ro-2-1-0-op-fc-im-112
- New forwarding PromptForBequeathingDataCausesTransferOfListOfSubscriptionsForEmbeddingStatusChanges (ro-2-1-0-op-fc-im-113) has been added

OAS:
- New callback PromptForBequeathingDataCausesTransferOfListOfSubscriptionsForEmbeddingStatusChanges has been added
